### DISCUSSION TITLE
Display article as in-app contextual help

### DISCUSSION THREAD

**Discussion by joarobles** [facebook/docusaurus#8433] [Q&A]
I have my Docusaurus site working perfectly and I wondered if I could use it to display in-app contextual help in my related webapp.

The general idea is to show an iframe in my webapp section with the related docusaurus article but hiding header, sidebar and footer (just the main article content). Something like this:

![image](https://user-images.githubusercontent.com/5776963/207122315-6b8563f1-c7de-46e4-b2a6-2fd780c2e34a.png)

I'd like to avoid deploying another docusaurus instance with a custom template and styling just to do this.

Is it possible? Thanks!

**Comment by slorber** [comment]
Hi

It's probably possible but we don't provide any first class feature to make it easy

Track https://github.com/facebook/docusaurus/issues/7480 for progress on this

**Comment by joarobles** [comment]
Thanks, I'll take a look ai it!




 *reweb*.com.ar Ing. Joaquín L. Robles
 +54 351 6111 350
 ***@***.***
